src/commonControls/listControl/ListControlAPL.ts

```diff
@@ -32,7 +33,7 @@ export namespace ListControlAPLPropsBuiltIns {
          *
          * Default: returns the value unchanged.
          */
-        valueRenderer: (value: string, input: ControlInput) => string;
+        valueRenderer: (value: string, input: ControlInput) => ListControlRenderedItem;
```

Comment:
There are probably a few breaking changes like this. please ensure the Commit Msg has them listed for inclusion in release notes.

Reply:
Added breaking change commit according the conventional commit rules.